modules/sdk-core/src/account-lib/baseCoin/enum.ts

```diff
@@ -42,9 +42,9 @@ export enum TransactionType {
   // Claim the staking rewards
   StakingClaim,
   // Add delegator to p-chain
-  addDelegator,
+  AddDelegator,
   // Add validator to p-chain
-  addValidator,
+  AddValidator,
```

Comment:
This is a breaking change

Reply:
You are right. I must update the commit.